index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Fobadara's Portfolio</title>
+    </head>
+    <body>
+        <!-- If section or article is used here linter error displays missing head tag hence the use of div -->
+        <div class="main-container">
+            <nav class="navigation" id="top-nav">
+                <div class="paragraph" id="welcome">Welcome</div>
+                <div id="hamburger"><img src="images/ham.svg" alt=""></div>
+            </nav>
+            <header class="header-container" id="top-header">
```

Comment:
- [ ] OPTIONAL
As specified kindly ensure that this section is enclosed with a `section` tag